corehq/apps/sms/models.py

```diff
@@ -290,6 +290,42 @@ def requeue(self):
             self.delete()
             queued_sms.save()
 
+    @classmethod
+    def get_counts_by_date(cls, domain, start_date, end_date, time_zone):
```

Comment:
Also nit: These could be static methods.

Reply:
updated: https://github.com/dimagi/commcare-hq/pull/20178/commits/c285a22d577274fb8633a177bc676f10916620a7